notebooks/oggm-tuto/merging_glaciers.ipynb

```diff
@@ -193,8 +181,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# use a suitable border size for your domain\n",
-    "cfg.PARAMS['border'] = 80\n",
+    "cfg.PARAMS['use_intersects'] = False\n",
```

Comment:
This is a problem, right?

Reply:
Not sure why it should be. I'll have to test